src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -125,6 +130,13 @@ class TransportCcEngine(
                 }
             }
         }
+        if (missingPacketDetailSeqNums.isNotEmpty()) {
+            logger.warn("TCC packet contained sequence numbers:\n" +
+                "${tccPacket.iterator().asSequence().map(PacketReport::seqNum).joinToString()}\n" +
+                "Couldn't find packet detail for the seq nums: \n${missingPacketDetailSeqNums.joinToString()}\n" +
+                "Oldest known seqNum was $oldestKnownSeqNum.")
```

Comment:
So this prints "Oldest known seqNum was null" if `sentPacketDetails` is empty?  A little weird, but I guess it's fine.

Reply:
right, i didn't think it was worth checking for that case just to make the print a little nicer (`no known sequence numbers` or whatever)

Reply:
Since we erase packet details from the map when we get feedback reports, the case that can happen is that you've already gotten reports on every packet you've sent, and then get a repeated TCC feedback message.  But I think this is fine.

Reply:
yeah, and it would be nice to be able to distinguish this case.  i wonder if there's a way we can without having to hold too much state.

Reply:
`"Couldn't find packet detail for the seq nums:\n${missingPacketDetailSeqNums.joinToString()}\n" +
(if (oldestKnownSeqNum != null) "Oldest known seqNum was $oldestKnownSeqNum.") else "sent packet details map was empty")`